test/rubocop_test.rb

```diff
@@ -18,10 +18,21 @@
 require './test/test_helper'
 
 class RubocopTest < Minitest::Test
-  def test_no_offenses_in_tests
-    output = `rubocop -f c -S --parallel ./test/*`
-    /[0-9]* files inspected,\ (.*)\ offenses\ detected/ =~ output
-    n_offeses = Regexp.last_match(1)
-    assert_equal('no', n_offeses, "Following new Rubocop offenses added to the tests:\n\n#{output}")
+  def test_no_error_in_files
+    parallel = ENV['RUBOCOP_PARALLEL'] || ENV['TRAVIS'] ? '--parallel' : nil
+    # parallel option could cause not to use rubocop from bundle
```

Comment:
In any case we can erase `--parallel` completely because on local it doesn't make any difference, since there is a cache and it only runs the modified files and on Travis there is only one thread..

Reply:
You should not have "red lines" for only convention offenses: check your linter to have a lighter visual indication for convention/refactor offenses. Other levels (Warning, Error, Fatal) should be linted with a more visual indication (for instance "red line" for Error/Fatal) and none offenses from theses types should be accepted. By definition, conventional offenses are less serious.